src/metrology_apis/__init__.py

```diff
@@ -32,7 +110,34 @@ def __rtruediv__(self, other: Self, /) -> Self: ...
 
 
 @runtime_checkable
-class Quantity[V, U: Unit](Protocol):
+class Quantity[V, U: UT, D: Dimension](Protocol):
```

Comment:
like this @jorenham ?

```suggestion
class Quantity[V = Any, U = Any, D = Any](Protocol):
```